randovania/data/json_data/prime3.txt

```diff
@@ -3284,19 +4070,25 @@ Asset id: 18200553231954089224
 
 > Pickup (Ship Grapple); Heals? False
   * Pickup 73; Major Location? True
+  > Samus Ship
+      Trivial
+
+> Samus Ship; Heals? True
   > Dock to Landing Access
       Trivial
+  > Pickup (Ship Grapple)
+      Trivial
 
 ----------------
 Botanica
```

Comment:
For Botanica, bottom level and top level aren't needed. Just have one helper node called Middle Level.

Save station b connects to chozo observatory as trivial.

Chozo observatory connects to middle level with lasso.

Middle level connects to broken lift with lasso.

Broken lift connects to middle level as trivial and missile with sa.

Missile connects to middle as trivial.